grafana-plugin/src/models/alert_receive_channel/alert_receive_channel.ts

```diff
@@ -149,6 +149,7 @@ export class AlertReceiveChannelStore {
 
     this.fetchCounters();
 
+    // @ts-ignore for some reason ts doesn't see `results` as list of ApiSchemas['AlertReceiveChannel']
```

Comment:
Let's not use ts-ignore unless we're missing backend types. You could easily get around by casting it to `ApiSchemas['AlertReceiveChannel'][]`. Didn't look closely, maybe it's related to the readonly fields in which case you'd need to rely on `OmitReadonlyMembers`

Reply:
ok, I can do `return results as Array<ApiSchemas['AlertReceiveChannel']>;`  seems `OmitReadonlyMembers` is not a point in my case, @brojd any ideas?

Reply:
The response you actually get from oncall helper is `AlertReceiveChannelPolymorphic` and  that's why it sees it as a missmatch. It's totally fine to just cast it to `AlertReceiveChannel`

Reply:
@maskin25 let's go with `return results as Array<ApiSchemas['AlertReceiveChannel']>;` , we need to clarify with backend under which conditions it's possible to get from this endpoint `components['schemas']['FilterAlertReceiveChannel'];` and improve the types. For now this casting is ok and better than ts-ignore